modules/context/repo.go

```diff
@@ -422,10 +422,10 @@ func RepoIDAssignment() func(ctx *Context) {
 }
 
 // RepoAssignment returns a middleware to handle repository assignment
-func RepoAssignment(ctx *Context) (cancel context.CancelFunc) {
```

Comment:
Side note, we should also enable `nonamedreturns` later

Reply:
well that will by fare conflict way more ...

Reply:
... one bit by one bit

Reply:
I'm against enabling it.
They are not a problem as long as there are no naked returns.
Instead, they rather help in cases where it is unclear what each of the return values mean

Reply:
thread opened in discord to talk about it :)

Reply:
Keep the named return value (`func foo() (canRead, canWrite bool)`) and use full return statement (`return canRead, canWrite` or `return false, false`)

Reply:
well for a cancleFunction to not name it cancle is kinda ok :D

Reply:
Yup, for this case, it's fine.

But for other cases, IMO we should keep using the named return value.